docs/wiki/01 - STUDY PARAMETERS SETUP/01 - Study Variables/04 - Subprojects.md

```diff
@@ -0,0 +1,46 @@
+# Subprojects
+
+## Overview
+**Subprojects** are defined in the front-end by the Configuration module, and are stored in the `subproject` table of the database.
+The **Subproject** plays an important role in determining the instrument battery that will be assigned to a candidate at a timepoint.  The subproject can be used as a variable when determining which instruments are populated -- this allows for subproject-specific batteries to be defined in the `test_battery` table. 
+
+## Adding Subproject Options
+
+### Front End (Recommended)
+Subprojects are defined in the Configuration module, which can be found in LORIS under the **Admin** menu tab.  Click on _To configure study subprojects click here_ link at the top of the page. Refer to the help section of the module for further instructions on how to add or modify subprojects.
+   
+### SQL
+Subprojects can be added directly in SQL using the following command.
+
+```sql
+INSERT INTO subproject (title) VALUES('SCI');
+```
+
+
+### API
+ _not yet available. See [API documentation](../../../API/) for latest additions_
+ 
+
+## Interaction With LORIS
+
+### Projects
+**Subprojects** must be associated to at least one Project in order to be able to create timepoints for candidates. This association should be defined directly on the front end through the Configuration module.
+
+1. Navigate to the Configuration module
```

Comment:
This isn't an interaction, this is just describing how the front end of the config module works..

Reply:
so help section ?